app/newRelicSnippet.tsx

```diff
@@ -0,0 +1,18 @@
+;NREUM.loader_config={accountID:"3340026",trustKey:"3340026",agentID:"1385887337",licenseKey:"NRJS-94c48cdf4bf4fa5718d",applicationID:"1385887337"}
+;NREUM.info={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",licenseKey:"NRJS-94c48cdf4bf4fa5718d",applicationID:"1385887337",sa:1}
```

Comment:
Would we want keys between local/deploy-preview and prod to be different?

Reply:
yea I wasn't sure about that. 
tbh I don't think it matters too much

Reply:
IMO we should. Can be as simple as not rendering the script locally?

Reply:
Whats the best way to do that?

Is there an inbuilt way or should I use env variables?

Reply:
@theramis just use Env variables. Probably can check if it's production at this point

Reply:
done